iam/contrib/iam_migration/management/commands/iam_makemigrations.py

```diff
@@ -61,8 +61,9 @@ def handle(self, *args, **options):
         sys.stdout.write("last migration: {}\n".format(last_migration_name))
 
         migration_name = self.migration_name(last_migration_name)
+        app_label = getattr(IAMMigrationConfig, "label", IAMMigrationConfig.name.rpartition(".")[2])
```

Comment:
this line will fetch IAMMigrationConfig's `app_label` directly.
It will cause a problem when user using this sdk in site-package way, the migration file will be created at site-package directory instead of user's project directory.
So the `MIGRATION_APP_NAME` must be configurable, let user determine the location of migration python file.

Reply:
> It will cause a problem when user using this sdk in site-package way, the migration file will be created at site-package directory instead of user's project directory.

Yeah, since pip installation need `BK_IAM_MIGRATION_APP_NAME` according to `docs/usage.md -> IAM Migration -> Django Migration`. If not setting `BK_IAM_MIGRATION_APP_NAME`, it will fail for `apps.get_app_config` since fallback to get `APP_NAME` for `MIGRATION_APP_NAME`:
```
(base) root@69faeb96:/tmp/123321/my_project# /tmp/venv/bin/python  -m django --version
2.2.24
(base) root@69faeb96:/tmp/123321/my_project# /tmp/venv/bin/python manage.py iam_makemigrations -v3 initial.json
is initial migration: True
last migration: None
Traceback (most recent call last):
  File "/tmp/venv/lib/python3.7/site-packages/django/apps/registry.py", line 155, in get_app_config
    return self.app_configs[app_label]
KeyError: 'iam.contrib.iam_migration'
```
 Maybe add check for `BK_IAM_MIGRATION_APP_NAME` for installation `bk-iam` with pip installation. So:
* pip installation: since `BK_IAM_MIGRATION_APP_NAME` exist after checking, migrations will in user app directory
* bundle src to app: since not pip installation, will use `iam.contrib.iam_migration` related directory

Reply:
Add check is a good idea, maybe it can prompt user to set `BK_IAM_MIGRATION_APP_NAME` when user install bk-iam through pip.
The bug of  source code bundle have been solve by https://github.com/TencentBlueKing/iam-python-sdk/pull/50

Reply:
OK. I will add check and update directly this PR.

Reply:
It's required to pay attention to different sitepackages directory in different OS Distro.(e.g.: Debian/Ubuntu)
```
>>> import site
>>> site.getsitepackages()
['/usr/local/lib/python3.6/dist-packages', '/usr/lib/python3/dist-packages', '/usr/lib/python3.6/dist-packages']
>>> import iam
>>> iam.__file__
'/usr/local/lib/python3.6/dist-packages/iam/__init__.py'

root@5dd72f555221:/# python3 -m venv /tmp/venv
>>> import site
>>> site.getsitepackages()
['/tmp/venv/lib/python3.6/site-packages', '/tmp/venv/local/lib/python3.6/dist-packages', '/tmp/venv/lib/python3/dist-packages', '/tmp/venv/lib/python3.6/dist-packages']
>>> import iam
>>> iam.__file__
'/tmp/venv/lib/python3.6/site-packages/iam/__init__.py'
```